cognite/powerops/prerun_transformations/transformations.py

```diff
@@ -874,7 +897,8 @@ def apply(self, time_series_data: tuple[pd.Series]) -> pd.Series:
         ```
         """
         single_ts = time_series_data[0]
-        if single_ts.empty:
+        if single_ts.empty or self.discharge.empty:
```

Comment:
I think it should work even if we have no inflow time series:
```suggestion
        if single_ts.empty:
            single_ts = pd.Series(data={start_time: 0})
        if self.discharge.empty:
```